torch/utils/cpp_extension.py

```diff
@@ -1045,6 +1048,12 @@ def _add_compile_flag(self, extension, flag):
         else:
             extension.extra_compile_args.append(flag)
 
+    def _hipify_compile_flags(self, extension):
+        # Simple hipify, map CUDA->HIP
+        if isinstance(extension.extra_compile_args, dict):
+            extension.extra_compile_args['nvcc'] = [
+                flag.replace("CUDA", "HIP") for flag in extension.extra_compile_args['nvcc']]
```

Comment:
I think it would be good to have log/print visibility of these hipify actions: log, print to console, which flags where modified. It would lend to better understanding on what hipify is doing and also track problems if these goes awry.

Reply:
@naromero77amd  I have opend a PR that merges the log changes and changes that prevent breaking existing flags by targetting in more detail when to do replacement vs existing replace all no matter what approach. 

https://github.com/ROCm/pytorch/pull/1964

Reply:
merged